standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+
+## 2. Introduction
+Possession of a trust domain's bundle is required in order to validate SVIDs from it. Therefore, achieving SPIFFE federation necessitates the exchange of SPIFFE bundles between trust domains. This exchange occurs on a regular basis, allowing the contents of a trust domain's bundle to change over time.
+
+To achieve this, SPIFFE federation defines a "bundle endpoint", which is a URL that serves up a SPIFFE bundle for a particular trust domain. Also defined are a set of "endpoint profiles", which specify the protocol and authentication semantics used between bundle endpoint servers and clients. Finally, this document further specifies the behavior of bundle endpoint clients and servers, and the management of federation relationships and the resulting bundle data.
+
+## 3. Targeted Use Cases
+Ultimately, SPIFFE Federation enables workloads to authenticate peers that reside in other trust domains. This functionality is necessary to support a wide range of use cases, however we wish to focus on three core use cases.
+
+SPIFFE Trust Domains are frequently used to segment environments with differing levels of trust within the same company or organization. Examples of this can be between staging and production, or between PCI and non-PCI environments. In these cases, the SPIFFE deployments used in each domain share a common administrative body and are likely backed by the same implementation. This is an important distinction, as it implies that the disparate deployments have the ability to agree on certain things (e.g. naming schemes), and that the security posture of each deployment can be known and well-understood by the others. 
+
+Second, SPIFFE Federation is also used between trust domains residing in different companies or organizations. This case is similar to the first in that we are federating between SPIFFE deployments, however due to potential differences in implementation and administration, coordination is generally limited to the data exchanged in the SPIFFE Federation protocol described herein.
+
+Finally, SPIFFE Federation can also enable use cases for consumers that don't yet have a mature SPIFFE control plane deployed. For example, a hosted product may wish to authenticate its customers using their SPIFFE identities without having to internally implement or deploy SPIFFE. This can be accomplished by allowing a workload to directly fetch the customer's trust domain bundle in order to authenticate their callers, negating the need to commit to a full-blown SPIFFE deployment.
```

Comment:
Hmm... the GitHub UI isn't showing me the text that I highlighted when I first created my comment. I'm not sure if that's a limitation of the UI or user error... in any case, this suggestion is to replace "negating" with "obviating".

Reply:
```suggestion
Finally, SPIFFE Federation can also enable use cases for consumers that don't yet have a mature SPIFFE control plane deployed. For example, a hosted product may wish to authenticate its customers using their SPIFFE identities without having to internally implement or deploy SPIFFE. This can be accomplished by allowing a workload to directly fetch the customer's trust domain bundle in order to authenticate their callers, obviating the need to commit to a full-blown SPIFFE deployment.
```